simtools/db_handler.py

```diff
@@ -25,3 +32,74 @@ def getArrayDB(databaseLocation):
     with open(file, 'r') as stream:
         out = yaml.load(stream, Loader=yaml.FullLoader)
     return out
+
+
+def getModelParameters(telescopeType, version, onlyApplicable=False):
+    '''
+    Read parameters from DB for one specific type (telescopeTYpe, site ...).
```

Comment:
Wrong doc. There is no site here. Site parameters should come from another function. Fixed!

Reply:
But this reminded me that there are no references to the site or telescope number. The DB will have South-LST-1 and North-MST-NectarCam-13. We probably need to provide the site/number, no?

Reply:
We can either treat South-LST-1, North-MST-NectarCam-13 as telescopeTypes or split in telescopeSize, site and number.
I think for the TelescopeModel is better to keep them as telescopeTypes. We can write an analogous function that receive as arguments telescopeSize, site and number as well, in case we need it in another context.

Reply:
At the moment it seems silly to divide telescope by number, but I am afraid it might be very necessary in a few years. So we need to think about it while we write code now. At this stage we shouldn't worry, but once the DB is incorporated, we need to discuss it again and see if we don't change everywhere to be for individual telescope already.